mantidproject/mantid
# Workbench: Instrument rendering option now loads correctly

**Description of work.**

Changed InstrumentWidgetDecoder to use int value for surface type. 
Instrument View now keeps the viewing angle it was saved with.

**To test:**
1. Open Instrument view in workbench
2. Change the viewing angle in the render tab to something other than Full 3D
3. Save the project
4. Close workbench
5. Open workbench
6. Open the project
7. Viewing angle should be the same as it was saved with. 

Fixes #26468 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
